landoapi/transplants.py

```diff
@@ -355,6 +356,15 @@ def warning_code_freeze(*, repo, **kwargs):
         ]
 
 
+@RevisionWarningCheck(9, "Revision has unresolved comments.")
+def warning_unresolved_comments(*, phab, revision, **kwargs):
+    if False in [
+        PhabricatorClient.expect(inline, "fields", "isDone")
+        for inline in get_inline_comments(phab, f"D{revision['id']}")
+    ]:
```

Comment:
This is the more Pythonic way:

```python
if not all(
    phab.expect(inline, "fields", "isDone")
    for inline in get_inline_comments(phab, f"D{revision['id']}")
):
```
It uses the [`all` builtin](https://docs.python.org/3/library/functions.html#all) which is more efficient and takes an iterator, so we don't need to create a full `list` - if one of the first few elements is `False` the function will return `False` immediately.

Nit: please use `phab.expect` here since we already have an instance of the class available.